### ISSUE TITLE
Add syntax to disable useless reporting for certain line-switches

### ISSUE THREAD

**Issue by ajafff** [fimbullinter/wotan#513]
From #358:

> there should be a way to say: "I know this chunk of code is nasty (or auto-generated), but I cannot fix it right now. I don't want any findings for it now and in the future."
>  * currently the disable comment is reported as useless if there are no findings with the current configuration
>*  possible solution: something with an exclamation mark in the comment

JS minifiers have special handling to not remove comments starting with an exclamation mark: `/*! content */`.
Maybe this could be adapted to mean: this comment is here on purpose and should not be reported by `--report-useless-directives`. To still allow for removal of such comments on minification the exclamation mark should not be placed at the beginning.

This should only prevent the comment from being reported as unused. However it should still be reported as useless if it's redundant.
It has no effect on enable comments.

```ts
// wotan-disable!
// wotan-enable!
// wotan-disable-line! rule1, rule2

// wotan-disable !
/* wotan-disable ! */ // this one is reported as useless because all rules are already disabled
```

Related: #357
